float64 duty_cycle      
float64 servo           
int32   solver_status   

